tools/acrn-crashlog/README.rst

```diff
@@ -0,0 +1,124 @@
+ACRN-Crashlog
+#############
+
+Introduction
+************
+
+ACRN-Crashlog is a debug tool to collect any useful logs/information after each crash/event occurred on the platform, including hypervisor, Service OS(SOS) and Android as a Guest(AaaG). It provides abundant logs for developers to debug/analyze the crashes. And it provides a flexible way to configure which crash/event to be concerned with xml file.
```

Comment:
Does the tool support collecting logs from Clearlinux running as a User OS (UOS)?

Reply:
Not yet. Currently, it supports Android as a Guest(AaaG). Need to modify the code to support Linux as a Guest(LaaG).